crates/turborepo-lib/src/run/mod.rs

```diff
@@ -323,19 +311,24 @@ impl<'a> Run<'a> {
 
         debug!("package file hashes: {:?}", package_file_hashes);
 
+        let spaces_client = self.base.spaces_api_client()?;
+        let spaces_api_auth = self.base.spaces_api_auth()?;
         let mut run_summary = RunSummary::new(
```

Comment:
What's the expected way for the task graph visitor to communicate to the run summary code since the run summary construction happens after the task graph gets visited?